bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -395,6 +406,61 @@ spec:
                 runAsNonRoot: true
                 runAsUser: 65534
               serviceAccountName: obo-prometheus-operator
+      - label:
+          app.kubernetes.io/name: prometheus-operator-admission-webhook
+          app.kubernetes.io/part-of: observability-operator
+          app.kubernetes.io/version: 0.60.0-rhobs1
+        name: obo-prometheus-operator-admission-webhook
+        spec:
+          replicas: 2
+          selector:
+            matchLabels:
+              app.kubernetes.io/name: prometheus-operator-admission-webhook
+              app.kubernetes.io/part-of: observability-operator
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                kubectl.kubernetes.io/default-container: prometheus-operator-admission-webhook
+              labels:
+                app.kubernetes.io/name: prometheus-operator-admission-webhook
+                app.kubernetes.io/part-of: observability-operator
+                app.kubernetes.io/version: 0.60.0-rhobs1
+            spec:
+              affinity:
+                nodeAffinity:
+                  preferredDuringSchedulingIgnoredDuringExecution:
+                  - preference:
+                      matchExpressions:
+                      - key: node-role.kubernetes.io/infra
+                        operator: Exists
+                    weight: 1
+              automountServiceAccountToken: false
+              containers:
+              - args:
+                - --web.enable-tls=true
+                - --web.cert-file=/tmp/k8s-webhook-server/serving-certs/tls.crt
+                - --web.key-file=/tmp/k8s-webhook-server/serving-certs/tls.key
+                image: quay.io/rhobs/obo-admission-webhook:v0.60.0-rhobs1
+                name: prometheus-operator-admission-webhook
+                ports:
+                - containerPort: 8443
+                  name: https
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 200Mi
+                  requests:
+                    cpu: 50m
+                    memory: 50Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  readOnlyRootFilesystem: true
+                terminationMessagePolicy: FallbackToLogsOnError
+              securityContext:
+                runAsNonRoot: true
+                runAsUser: 65534
```

Comment:
I'm not 100% sure but couldn't we drop this? This would only be important if the admission webhook would access the PV. This way it would also simplify the deployment since we could also stop deploying the cluster-role & role-binding